src/App/Layout.tsx

```diff
@@ -14,13 +15,22 @@ const { HEADER_LINKS, GROUPS_DATA, SOCIAL_MEDIA_LINKS } = IS_AST
 
 export default function Layout() {
   return (
-    <div className="grid grid-rows-[auto_1fr_auto]">
+    <div className="grid grid-rows-[auto_auto_1fr_auto]">
       <Seo /> {/* Load all defaults for SEO meta tags */}
-      <Header
-        classes="sticky top-0 z-20"
-        links={HEADER_LINKS}
-        logo={LOGO_DARK}
-      />
+      <div className="top-0 text-center dark:bg-blue bg-blue-d3">
+        <span className="md:text-sm md:uppercase text-white">
+          Looking for the old site? It has moved to{" "}
+          <ExtLink
+            key="banner-link-legacy"
+            className="transition ease-in-out duration-300 hover:text-orange-l1"
+            href="https://legacy.angelgiving.io"
+          >
+            https://legacy.angelgiving.io
+          </ExtLink>{" "}
+          and will be sunset on Oct 13, 2023. Migrate to Polygon today!
+        </span>
+      </div>
+      <Header classes="sticky z-20" links={HEADER_LINKS} logo={LOGO_DARK} />
```

Comment:
```suggestion
      <Header classes="top-0 sticky z-20" links={HEADER_LINKS} logo={LOGO_DARK} />
```

`top-0` should stay in `Header` so that it remains to stick on top of the page
<img width="1469" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/77395111-8f63-42f9-8556-ace22566595d">

this would also mean, `Notice` is hidden when user scrolls down